dev/Common/TerminalVelocityFeatures-DynamicDependency.h

```diff
@@ -3,21 +3,21 @@
 
 // THIS FILE IS AUTOMATICALLY GENERATED; DO NOT EDIT IT
 
-// INPUT FILE: dev\Common\TerminalVelocityFeatures-DynamicDependency.xml
-// OPTIONS: -Channel Experimental -Namespace Microsoft.Windows.ApplicationModel.DynamicDependency
+// INPUT FILE: dev\\Common\\TerminalVelocityFeatures-DynamicDependency.xml
+// OPTIONS: -Channel Experimental -Language C++ -Namespace Microsoft.Windows.ApplicationModel.DynamicDependency
 
 // Feature constants
-#define WINDOWSAPPRUNTIME_MICROSOFT_WINDOWS_APPLICATIONMODEL_DYNAMICDEPENDENCY_FEATURE_GENERATIONID_ENABLED 1
+#define WINDOWSAPPRUNTIME_MICROSOFT_WINDOWS_APPLICATIONMODEL_DYNAMICDEPENDENCY_FEATURE_BOOTSTRAPCPPINITIALIZATION_ENABLED 1
 
 #if defined(__cplusplus)
 
 namespace Microsoft::Windows::ApplicationModel::DynamicDependency
```

Comment:
Can these APIs be under the winrt namespace when using alongside C++/WinRT so that you don't have any namespace collisions? I know it's not a WinRT but still, consider keeping it under namespace winrt for consistency.

Reply:
When would you have namespace collisions?

These aren't WinRT APIs so it'd be pretty odd for them to appear in a winrt:: namespace.

Hmmm. These are primarily used in implementation of a feature, not using a feature. The header would appear to users if a Flat-C API needed the #define'd constants, but in that case the __cplusplus block isn't relevant. Only reason someone using WinAppSDK would need the that is if a feature provided a header with inline code and that needed to be compiled in/out.

It should be possible to omit the __cplusplus block in the common case e.g.

```
#if defined(__cplusplus) && defined(WINDOWSAPPRUNTIME_MICROSOFT_..._CPLUSPLUS_ENABLED)
```

Thanks for bringing this up